### PR TITLE
move user agent to registry

### PR DESCRIPTION
**Pull request by trisch-me**
## Changes

Please provide a brief description of the changes here.
- Moving user agent to the registry
- ~Adding 3 new fields from ECS to the registry~ deferred to follow-up PR

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] [CHANGELOG.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md) updated for non-trivial changes.
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by joaopgrassi — APPROVED** [review]

**Review by joaopgrassi — CHANGES_REQUESTED** [review]
Almost missed it. Please split the PR and send one to add the ECS ones separately. Let's keep moving the attributes to registry a "simple" change.

File: model/registry/user-agent.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        type: string
+        brief: >
+          Name of the device extracted from original
+        examples: ['iPhone']
+      - id: name
+        type: string
+        brief: >
+          Name of the user agent extracted from original
+        examples: ['Safari']
+      - id: version
```
Comment (jsuereth) [inline model/registry/user-agent.yaml:23]: What should this be in the event of multiple versions?

File: model/registry/user-agent.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        brief: >
+          Value of the [HTTP User-Agent](https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent) header sent by the client.
+        examples: ['CERN-LineMode/2.15 libwww/2.17b3',
+                   'Mozilla/5.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Mobile/15E148 Safari/604.1']
+      - id: device_name
+        type: string
+        brief: >
+          Name of the device extracted from original
+        examples: ['iPhone']
+      - id: name
```
Comment (jsuereth) [inline model/registry/user-agent.yaml:18]: What should this be in the event of multiple names?  User-Agent tends to be "additive" w/ layers of software, and isn't just browser-stacks.

**Review by AlexanderWert — APPROVED** [review]

**Comment by AlexanderWert** [comment]
nit: @trisch-me please also change the title

**Review by trask — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]

File: model/registry/user-agent.yaml (hunk 1 above)
Comment (trisch-me) [reply model/registry/user-agent.yaml]: Let's move this discussion to the new PR after merging this one

**Pull request merged**